docker-compose.yml

```diff
@@ -5,8 +5,6 @@ services:
     build: .
     image: aeternity/aeternity:${IMAGE_TAG-latest}
     hostname: node1
-    environment:
-      EPOCH_CONFIG: /home/aeternity/aeternity.yaml
```

Comment:
Please rebase on recent master.
I am concerned that now-merged paraller PR #2057 may affect result in master.